.gitignore

```diff
@@ -3,19 +3,21 @@
 # Files and directories created by pub
 .dart_tool/
 .packages
-build/
+
 # If you're building an application, you may want to check-in your pubspec.lock
 pubspec.lock
 
-# Directory created by dartdoc
-# If you don't generate documentation locally you can remove this line.
-doc/api/
+# Conventional directory for build outputs.
+build/
+
+# Omit commiting executable file, will remove before merging
+bin/
```

Comment:
this line did not remove the bin dir? so remove also the main file?

Reply:
thats weird. I removed this since we need the `bin/` and its contents, also the executable will be ignored separately since we are ignoring `.exe` files already.

Reply:
we can accept the executable for this case, we are building a template, so this is the job of the template implementor :)